test/acceptance/keywords/elements.robot

```diff
@@ -69,3 +69,9 @@ Get Vertical Position
     Should Be True    ${pos} > ${0}
     Run Keyword And Expect Error    Could not determine position for 'non-existent'
     ...    Get Horizontal Position    non-existent
+
+Get Element Size
+	${width}  ${height}=  Get Element Size  link=Link
+	Should be True  ${height} > ${0}
+	Should be True  ${width} > ${0}
+	Run Keyword And Expect Error  Could not determine size for 'non-existent'  Get Element Size  non-existent
```

Comment:
1 I think line 77 is not needed
2 Write test for where element has size put it is hidden. 

Example:
```
| Element Should Not Be Visible | locator=here |
| ${width} | ${height} = | Get Element Size | locator=here | 
| Verify | ${width} | ${height} | # No idea what value actually will be | 
```
3 Write test for element that should not have size. Example for a ```<div></>``` which does not contain anything
4 Add new line in the end.